apps/golang/install

```diff
@@ -4,26 +4,18 @@ VERSION="1.19.2"
 [ -z "$GOROOT" ] && GOROOT="$HOME/.go"
 [ -z "$GOPATH" ] && GOPATH="$HOME/go"
 
-OS="$(uname -s)"
-ARCH="$(uname -m)"
-
-case $OS in
-    "Linux")
-        case $ARCH in
-        "aarch64")
-            ARCH=arm64
-            ;;
-        "armv6" | "armv7l")
-            ARCH=armv6l
-            ;;
-        "armv8")
-            ARCH=arm64
-            ;;
-        esac
-        PLATFORM="linux-$ARCH"
+case "$arch" in
+  "64")
+    ARCH="arm64"
     ;;
-
-esac
+  "32")
+    ARCH="armv6l"
+    ;;
+  *)
+    error "arch variable is not set, can not continue"
+    PLATFORM="linux-$ARCH"
```

Comment:
incorrect location for setting the `PLATFORM` variable. look at the previous code again

Reply:
@theofficialgman Better in b8d7f978380379edf56856517411790d3bbd1cf7?